tests/e2e/logging/test_openshift-logging.py

```diff
@@ -132,11 +146,15 @@ def test_create_new_project_to_verify_logging(self, create_pvc_and_deploymentcon
         elasticsearch_pod_obj = get_pod_obj(
             name=elasticsearch_pod[1], namespace='openshift-logging'
         )
-        projects = elasticsearch_pod_obj.exec_cmd_on_pod(
-            command='indices | grep project', out_yaml_format=True
-        )
-        logger.info(projects)
-        if pvc_obj.project.namespace in projects:
-            logger.info("The new project exists in the EFK stack")
-        else:
-            raise ModuleNotFoundError
+
+        @retry(ModuleNotFoundError, 10, 200, 3)
```

Comment:
this function doesn't have to be nested and can be defined outside of parent method. You just need to pass reference to elasticsearch_pod_obj and pvc_obj.

Reply:
She asked me about it before, it was a way to avoid the @retry as it needs a function name, the function itself is not used anywhere, so it being nested doesn't really matter much.